Header header
int16 reward
float32[] observations
float32[] prev_state
bool final_state